### PR TITLE
[hotfix] [docs] Fix typo in checkpoint monitoring doc

### PR DESCRIPTION
**Pull request by brandonJY**
fix typo in checkpoint_monitoring doc

### CODE REVIEW

File: docs/monitoring/checkpoint_monitoring.zh.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The checkpoint history keeps statistics about recently triggered checkpoints, in
 - **Trigger Time**: The time when the checkpoint was triggered at the JobManager.
 - **Latest Acknowledgement**: The time when the latest acknowledged for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
 - **End to End Duration**: The duration from the trigger timestamp until the latest acknowledgement (or n/a if no acknowledgement received yet). This end to end duration for a complete checkpoint is determined by the last subtask that acknowledges the checkpoint. This time is usually larger than single subtasks need to actually checkpoint the state.
-- **Checkpointed Data Size**: The state size over all acknowledged subtasks.
+- **Checkpointed Data Size**: The checkpointed data size over all acknowledged subtasks. This value would be the delta delta checkpointed data if incremental checkpoint is on.
```
Comment (zentol) [inline docs/monitoring/checkpoint_monitoring.zh.md:64]:
```suggestion
- **Checkpointed Data Size**: The checkpointed data size over all acknowledged subtasks. If incremental checkpointing is enabled this value is the checkpointed data size delta.
```

File: docs/monitoring/checkpoint_monitoring.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ The checkpoint history keeps statistics about recently triggered checkpoints, in
 - **Trigger Time**: The time when the checkpoint was triggered at the JobManager.
 - **Latest Acknowledgement**: The time when the latest acknowledged for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
```
Comment (XComp) [inline docs/monitoring/checkpoint_monitoring.md:62]: Could you also fix that one while reiterating over the file, please:
```suggestion
- **Latest Acknowledgement**: The time when the latest acknowledgement for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
```

File: docs/monitoring/checkpoint_monitoring.zh.md (hunk 1 above)
Comment (brandonJY) [reply docs/monitoring/checkpoint_monitoring.zh.md]: @zentol  updated per suggested

File: docs/monitoring/checkpoint_monitoring.md (hunk 2 above)
Comment (brandonJY) [reply docs/monitoring/checkpoint_monitoring.md]: Thanks for the review. Updated per suggested.

**Pull request merged**